README.md

```diff
@@ -162,6 +162,9 @@ In case you need to assign a specific port to a Bypass instance to listen on, yo
 bypass = Bypass.open(port: 1234)
 ```
 
+> Note: `Bypass.open/0` **cannot** be called in a `setup_all` blocks due to the way Bypass verifies the expectations, at the end of each
```

Comment:
Can you say `must not`?

And I don't think you need the `,` in `, at the end of each test`.